From 8cf4212ea40e1b843e1aecf4b24681b0964ec04c Mon Sep 17 00:00:00 2001
From: Tim Buckingham <mr@timbuckingham.com>
Date: Fri, 7 Apr 2017 15:11:03 -0400
Subject: [PATCH] Fixing empty space at the end of a file name allowing
 dangerous files to be uploaded. Thank you to math1as from l-team for the
 disclosure and fix.

https://github.com/bigtreecms/BigTree-CMS/issues/276
---
 core/inc/bigtree/apis/storage.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/inc/bigtree/apis/storage.php b/core/inc/bigtree/apis/storage.php
index e28ae5477..0c867f8c1 100755
--- a/core/inc/bigtree/apis/storage.php
+++ b/core/inc/bigtree/apis/storage.php
@@ -8,7 +8,7 @@ class BigTreeStorage {
 
 		var $AutoJPEG = false;
 		var $DisabledFileError = false;
-		var $DisabledExtensionRegEx = '/\\.(exe|com|bat|php|rb|py|cgi|pl|sh|asp|aspx)$/i';
+		var $DisabledExtensionRegEx = '/\\.(exe|com|bat|php|rb|py|cgi|pl|sh|asp|aspx)/i';
 		var $Service = "";
 		var $Cloud = false;
 		var $Settings;
